ajax/libs/ammaps/package.json

```diff
@@ -1,14 +1,39 @@
 {
   "name": "ammaps",
   "filename": "ammap.js",
-  "version": "3.13.0",
```

Comment:
Nothing has changed since last review, this version removal still needs to be reverted.